release/models/isis/openconfig-isis.yang

```diff
@@ -925,7 +931,61 @@ module openconfig-isis {
         graceful restart procedures during its own restart, but supports
         retaining forwarding information during a remote speaker's restart.";
     }
-    reference "RFC 5306: Restart Signaling for IS-IS.";
+
+    leaf non-planed-only {
+      type boolean;
+      description
+        "When this leaf is set to TRUE, planned restart procedures, as
+        described in RFV8706 are not used.";
+      reference
+      "RFC 5706: Restart Signaling
+      for IS-IS";
+    }
+
+    reference
+      "RFC 5306: Restart Signaling for IS-IS; RFC 5706: Restart Signaling
+      for IS-IS";
+  }
+
+  grouping isis-graceful-restart-level-config {
+    description
+      "This grouping defines ISIS graceful restart configuration relevant
+      for ISIS level/LSDB";
+
+    leaf restart-time {
+      type int64;
```

Comment:
```suggestion
      type unit16;
```

RFC says it's 2 octets on the wire, so max value couldn't be higher than 16 bits?  Also made this unsigned since there's not a negative time here.